godtools/App/Services/Renderer/Views/MobileContent/Views/Page/MobileContentPageViewModel.swift

```diff
@@ -55,29 +52,40 @@ class MobileContentPageViewModel: MobileContentPageViewModelType {
             return nil
         }
         
-        let manifestAttributes: MobileContentManifestAttributesType = rendererPageModel.manifest.attributes
+        let manifest: Manifest = renderedPageContext.manifest
         
-        let backgroundImageModel: BackgroundImageModelType?
+        let backgroundImageModel: BackgroundImageModel?
         
-        if pageModel.backgroundImageExists {
-            backgroundImageModel = pageModel
+        if let pageBackgroundImageFileName = pageModel.backgroundImage?.name {
+            
+            backgroundImageModel = BackgroundImageModel(
+                backgroundImageFileName: pageBackgroundImageFileName,
+                backgroundImageAlignment: pageModel.backgroundImageGravity,
+                backgroundImageScale: pageModel.backgroundImageScaleType
+            )
         }
-        else if manifestAttributes.backgroundImageExists {
-            backgroundImageModel = manifestAttributes
+        else if let manifestBackgroundImageFileName = manifest.backgroundImage?.name {
+            
+            backgroundImageModel = BackgroundImageModel(
+                backgroundImageFileName: manifestBackgroundImageFileName,
+                backgroundImageAlignment: manifest.backgroundImageGravity,
+                backgroundImageScale: manifest.backgroundImageScaleType
+            )
         }
```

Comment:
I actually think you aren't fully handling background images correctly here. It's possible to define both a manifest and a page background and have them layered overtop of each other.

There isn't any actual usages in production that I'm aware of, but the manifest & page background logic is meant to be layered and not one or the other.

Reply:
this is a bigger change than the scope of this PR

Reply:
Hey @frett created this ticket here -> https://jira.cru.org/browse/GT-1485